ZipBuilder/ZipBuilderV2Plan.md

```diff
@@ -0,0 +1,58 @@
+V2 will add an option to generate a zip distribution of binary frameworks from an arbitrary list
+of source and binary CocoaPods.
+
+## Introduction
+
+The current [Zip Builder](https://github.com/firebase/firebase-ios-sdk/tree/master/ZipBuilder)
+is Firebase specific. This is a proposal and initial plan
+to evolve the Zip Builder into a multi-purpose Apple binary framework creation tool.
+
+It would be useful to have a generic
```

Comment:
Spacing seems wrong here, more can fit on this line

Reply:
Done.